app/models/stance.rb

```diff
@@ -14,12 +14,17 @@ def info
   end
 
   def self.search(words)
-    words.split(' ').map do |word|
-      Stance.all.select do |s|
-        position = Position.find(s.position_id)
-        issue = Issue.find(position.issue_id)
-        position.description.downcase.include?(word.downcase) || issue.description.downcase.include?(word.downcase)
-      end
-    end.flatten.uniq
+    case words
```

Comment:
should we downcase the words before running logic? what if they search 'popular'?